daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/types/Ledger.scala

```diff
@@ -66,9 +77,11 @@ object Ledger {
 
   case class ScenarioTransactionId(index: Int) extends Ordered[ScenarioTransactionId] {
     def next: ScenarioTransactionId = ScenarioTransactionId(index + 1)
+    // The resulting LedgerString is at most 11 chars long
     val id: LedgerString = LedgerString.fromLong(index.toLong)
     override def compare(that: ScenarioTransactionId): Int = index compare that.index
-    def makeCommitPrefix: LedgerString = LedgerString.concat(`#`, id, `:`)
+    // The resulting LedgerString is at most 12 chars long
```

Comment:
How can this string be at most 12 characters long, if it adds to characters and `id` can be at most 11 characters long?

Reply:
* `id` is an `Int` converted in `String` (at most 11 chars) 
* `#` is 1 char (added by rebase)
* `:` is 1 char 
total at most 13 (corrected)